components/Main/NewsletterSignUp/SignUpForm/SignUpForm.module.css

```diff
@@ -0,0 +1,18 @@
+.form > * {
+  border: 1px solid gray;
+  font-size: 0.9rem;
+  line-height: var(--spaceOneRatioBigger);
+  margin: 0;
+  padding: calc(var(--spaceThreeRatiosSmaller) - 1px) calc(var(--spaceTwoRatiosSmaller) - 1px);
+}
+
+.form > button:hover {
```

Comment:
This and the `input:focus` styles below are examples of where we could have a `button` and `input` class that had these styles in and then was just put on the element maybe.